src/wp-includes/deprecated.php

```diff
@@ -3340,7 +3340,18 @@ function gd_edit_image_support($mime_type) {
 function wp_convert_bytes_to_hr( $bytes ) {
 	_deprecated_function( __FUNCTION__, '3.6.0', 'size_format()' );
 
-	$units = array( 0 => 'B', 1 => 'KB', 2 => 'MB', 3 => 'GB', 4 => 'TB' );
+	$units = array(
+		/* translators: Unit symbol for byte. */
+		0 => _x( 'B', 'unit symbol' ),
+		/* translators: Unit symbol for kilobyte. */
+		1 => _x( 'KB', 'unit symbol' ),
+		/* translators: Unit symbol for megabyte. */
+		2 => _x( 'MB', 'unit symbol' ),
+		/* translators: Unit symbol for gigabyte. */
+		3 => _x( 'GB', 'unit symbol' ),
+		/* translators: Unit symbol for terabyte. */
+		4 => _x( 'TB', 'unit symbol' ),
+	);
```

Comment:
Since the function is deprecated it shouldn't get any new features.

Reply:
I was asking myself if it was usefull to improve it, reverting it back :)